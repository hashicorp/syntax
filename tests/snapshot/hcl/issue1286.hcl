variable "foo" {
  description = "bar"
  nullable    = true
  falseyy     = "bar"
  trueyy      = "bar"
  null        = "true"
  false       = "true"
  true        = "bar"
  nullable=true
  falseyy=""
  trueyy=""
  null=true
  false=""
  true=""
}
