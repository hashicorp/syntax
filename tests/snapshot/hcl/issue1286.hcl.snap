>variable "foo" {
#^^^^^^^^ source.hcl meta.block.hcl entity.name.type.hcl
#        ^ source.hcl meta.block.hcl
#         ^^^^^ source.hcl meta.block.hcl variable.other.enummember.hcl
#              ^ source.hcl meta.block.hcl
#               ^ source.hcl meta.block.hcl punctuation.section.block.begin.hcl
>  description = "bar"
#^^ source.hcl meta.block.hcl
#  ^^^^^^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#             ^ source.hcl meta.block.hcl variable.declaration.hcl
#              ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#               ^ source.hcl meta.block.hcl variable.declaration.hcl
#                ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                 ^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                    ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  nullable    = true
#^^ source.hcl meta.block.hcl
#  ^^^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#          ^^^^ source.hcl meta.block.hcl variable.declaration.hcl
#              ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#               ^ source.hcl meta.block.hcl variable.declaration.hcl
#                ^^^^ source.hcl meta.block.hcl constant.language.hcl
>  falseyy     = "bar"
#^^ source.hcl meta.block.hcl
#  ^^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#         ^^^^^ source.hcl meta.block.hcl variable.declaration.hcl
#              ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#               ^ source.hcl meta.block.hcl variable.declaration.hcl
#                ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                 ^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                    ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  trueyy      = "bar"
#^^ source.hcl meta.block.hcl
#  ^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#        ^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl
#              ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#               ^ source.hcl meta.block.hcl variable.declaration.hcl
#                ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                 ^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                    ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  null        = "true"
#^^ source.hcl meta.block.hcl
#  ^^^^ source.hcl meta.block.hcl constant.language.hcl
#      ^^^^^^^^^^ source.hcl meta.block.hcl
#                ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                 ^^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                     ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  false       = "true"
#^^ source.hcl meta.block.hcl
#  ^^^^^ source.hcl meta.block.hcl constant.language.hcl
#       ^^^^^^^^^ source.hcl meta.block.hcl
#                ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                 ^^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                     ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  true        = "bar"
#^^ source.hcl meta.block.hcl
#  ^^^^ source.hcl meta.block.hcl constant.language.hcl
#      ^^^^^^^^^^ source.hcl meta.block.hcl
#                ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#                 ^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                    ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  nullable=true
#^^ source.hcl meta.block.hcl
#  ^^^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#          ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#           ^^^^ source.hcl meta.block.hcl constant.language.hcl
>  falseyy=""
#^^ source.hcl meta.block.hcl
#  ^^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#         ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#          ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#           ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  trueyy=""
#^^ source.hcl meta.block.hcl
#  ^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#        ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#         ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#          ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  null=true
#^^ source.hcl meta.block.hcl
#  ^^^^ source.hcl meta.block.hcl constant.language.hcl
#      ^ source.hcl meta.block.hcl
#       ^^^^ source.hcl meta.block.hcl constant.language.hcl
>  false=""
#^^ source.hcl meta.block.hcl
#  ^^^^^ source.hcl meta.block.hcl constant.language.hcl
#       ^ source.hcl meta.block.hcl
#        ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#         ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  true=""
#^^ source.hcl meta.block.hcl
#  ^^^^ source.hcl meta.block.hcl constant.language.hcl
#      ^ source.hcl meta.block.hcl
#       ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#        ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>}
#^ source.hcl meta.block.hcl punctuation.section.block.end.hcl
>